modules/ROOT/pages/create-reference-map.adoc

```diff
@@ -0,0 +1,163 @@
+= Creating Reference Maps for Message Types
+
+When you create a new message type, you can associate it with a DataWeave reference map. This is mandatory for Receive from Partners message types formatted with JSON or XML and for all Source at Host message types.
```

Comment:
At the end, please add: "The reference map can also include custom message attributes that need to be extracted from the message payloads at runtime and make them available for visibility, tracking and search, for any Receive from partners or Source at host message types."